### ISSUE TITLE
Unable to find dataset with same class labels as used in the weights obtained from bvlc_alexnet.npy

### ISSUE THREAD

**Issue by pg1647** [kratzert/finetune_alexnet_with_tensorflow#56]
Can you point me in the direction where I can get a large dataset with class labels in the same format as used in bvlc_alexnet.npy? I downloaded ILSVRC2010 validation and test dataset but the ground truth labels found in the downloaded dataset from Imagenet website does not correspond to the right class labels in the neural network architecture used in this repository.

**Comment by kratzert** [comment]
No sorry I can't. I have the class file, as well as the weights from here [http://www.cs.toronto.edu/%7Eguerzhoy/tf_alexnet/](http://www.cs.toronto.edu/%7Eguerzhoy/tf_alexnet/) and never downloaded the "real" imagenet data set.
